hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -50,19 +51,35 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
           val institutionPersistence = sharding.entityRefFor(
             InstitutionPersistence.typeKey,
             s"${InstitutionPersistence.name}-${institution.LEI}")
+
+          val fInstitution
+            : Future[Option[Institution]] = institutionPersistence ? (
+              ref => GetInstitution(ref)
+          )
           timedPost { uri =>
-            val fCreated
-              : Future[InstitutionCreated] = institutionPersistence ? (ref =>
-              CreateInstitution(institution, ref))
-            onComplete(fCreated) {
-              case Success(InstitutionCreated(i)) =>
-                complete(ToResponseMarshallable(StatusCodes.Created -> i))
-              case Failure(error) =>
-                val errorResponse =
-                  ErrorResponse(500, error.getLocalizedMessage, uri.path)
+            onComplete(fInstitution) {
+              case Success(Some(_)) =>
+                val errorResponse = ErrorResponse(
+                  400,
+                  s"Institution ${institution.LEI} already exists",
+                  uri.path)
                 complete(ToResponseMarshallable(
-                  StatusCodes.InternalServerError -> errorResponse))
+                  StatusCodes.BadRequest -> errorResponse))
```

Comment:
You can use the utilities in `FilingResponseUtils` to make this look a bit cleaner